.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 88
+select = C,E,F,W,B,B950
+extend-ignore = E203, E501, W503
```

Comment:
why `E203` and `E501`?

Reply:
`E501`: disabled in favor of `B950`. Quote from link "will align more with black’s “try to respect --line-length, but don’t become crazy if you can’t”". [link](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length)
`E203`: Black has a different take on what "complex expressions" are and hence has different whitespace around the slice operator than E203 would want. [link](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#slices)